blockchains/ethereum/tokenlist.json

```diff
@@ -3642,7 +3642,7 @@
             "address": "0x4Fabb145d64652a948d72533023f6E7A623C7C53",
             "name": "Binance USD",
             "symbol": "BUSD",
-            "decimals": 18,
+            "decimals": 8,
```

Comment:
Why?

https://etherscan.io/token/0x4Fabb145d64652a948d72533023f6E7A623C7C53

decimals is 18